guides/mainnet-genesis/README.md

```diff
@@ -100,7 +100,8 @@ where:
 - `<your-rpc-url>` is the URL of the Ethereum RPC node you are using
 - `<your-operator-private-key>` is the private key of the operator account you are using
 - `<your-extra-data-string>` is any extra string you want to include in the registration,
-  such as your operator name, website or a custom identifier
+  such as your operator name, website or a custom identifier. If you use a single word, please
+  avoid using quotes (e.g. `my-operator` instead of `"my-operator"`).
```

Comment:
Some questions:
- can't we strip away `"` on the `bolt` CLI instead?
- I think "word" might be a little ambiguous since I don't consider a URL a word. Maybe you were referring about the presence of whitespaces
- can you make more examples of what is the on-chain result based on the input?